test/execution_summary_test.py

```diff
@@ -1086,3 +1086,47 @@ class Bar(RunOnceTask):
         self.assertIn('Luigi Execution Summary', s)
         self.assertNotIn('00:00:00', s)
         self.assertNotIn('\n\n\n', s)
+
+    def test_statuse_with_task_retry(self):
```

Comment:
@Tarrasch 
Hmm, I see.
I will try to do that.

@erikbern 
Oops.
Will fix:)